test/fast/commit-test.ts

```diff
@@ -28,8 +25,8 @@ describe('commit', () => {
     const result = await GitProcess.exec(['commit', '-F', '-'], testRepoPath, { stdin: message })
 
     verify(result, r => {
```

Comment:
How do you feel about using `expect(result).toMatchSnapshot()` instead here? That would assert that all the keys are consistent. There's also a `toMatchInlineSnapshot` method that integrates with Prettier to insert a template literal inside the parens so there doesn't have to be another file.

Reply:
Interesting, but I mostly wanted to focus on the flaky timeouts.

Feel free to move this idea to a fresh issue so we can track it.

Reply:
➡️ #236